deployment/environments/terraform-test.tfvars

```diff
@@ -21,6 +21,13 @@ rds_database_identifier = "opensupplyhub-enc-tst"
 rds_database_name = "opensupplyhub"
 rds_multi_az = false
 rds_storage_encrypted = true
+snapshot_identifier = "opensupplyhub-rds-tst"
+
+anonymized_database_kms_key_id = "arn:aws:kms:eu-west-1:249322298638:key/f311ecbd-a011-48a1-a676-8eeac63cd6d4"
```

Comment:
Maybe it is better to keep kms key in this repo: https://github.com/opensupplyhub/ci-deployment
cc: @killev

Reply:
I can move it, but it's just a key identifier, it's not sensitive information.

Reply:
Please, move it to https://github.com/opensupplyhub/ci-deployment repo. While the KMS key ID alone does not grant access to decrypt data, it can potentially reveal which resources are encrypted with a specific KMS key.

Reply:
@VadimKovalenkoSNF 
done